.github/workflows/zeebe-aws-os-dispatch-tests.yml

```diff
@@ -0,0 +1,109 @@
+name: Zeebe CI (AWS OS)
+
+on:
+  workflow_dispatch: { }
+  workflow_call: { }
+
+defaults:
+  run:
+    # use bash shell by default to ensure pipefail behavior is the default
+    # see https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#exit-codes-and-error-action-preference
+    shell: bash
+
+env:
+  DOCKER_PLATFORMS: "linux/amd64,linux/arm64"
+
+jobs:
+ integration-tests:
+    name: "[IT] ${{ matrix.name }}"
+    timeout-minutes: 15
+    runs-on: ${{ matrix.runs-on }}
+    strategy:
+      fail-fast: false
+      matrix:
+        group: [ os-exporter, camunda-exporter ]
+        include:
+          - group: os-exporter
+            name: "Zeebe OS Exporter Integration Tests"
+            maven-modules: "'io.camunda:zeebe-opensearch-exporter'"
+            maven-build-threads: 2
+            maven-test-fork-count: 7
+            runs-on: ubuntu-latest
+          - group: camunda-exporter
+            name: "Camunda Exporter Integration Tests"
+            maven-modules: "'io.camunda:camunda-exporter'"
+            maven-build-threads: 2
+            maven-test-fork-count: 7
+            runs-on: ubuntu-latest
```

Comment:
Using `runs-on: ubuntu-latest` for all jobs means that *none of those tests* are running against a real AWS OpenSearch instance - the real AWS OpenSearch instance (available under `CAMUNDA_CI_OPENSEARCH_2_9_URL`) is only reachable from Camunda-hosted runners but not from GitHub.

I'm not sure if it is intentional to not use a real AWS OpenSearch instance?